spec/springboard/client/uri_spec.rb

```diff
@@ -12,10 +12,38 @@
   end
 
   describe "parse" do
-    it "should return a URI based on the given string" do
-      uri = described_class.parse('/some_path')
-      expect(uri).to be_a(described_class)
-      expect(uri.to_s).to eq('/some_path')
+    describe "when called with a URI object" do
+      it "should return a cloned URI object" do
+        parsed_uri = described_class.parse(uri)
+        expect(uri.class).to eq(parsed_uri.class)
+        expect(uri.object_id).not_to eq(parsed_uri.object_id)
```

Comment:
Should add an expectation that the string representations are the same, too. Otherwise this test would still pass if we returned _any_ `URI` object.

Reply:
Makes sense, fixed.